src/CreateHandler/CreateHandler.php

```diff
@@ -37,10 +37,11 @@ public function __construct(string $classSkeleton = self::CLASS_SKELETON, string
     }
 
     /**
+     * @param string $class
```

Comment:
Please remove, it is redundant, doesn't give us any additional information than function signature. This is how we are doing now in ZF.